app/code/Magento/BraintreeGraphQl/etc/schema.graphqls

```diff
@@ -11,9 +11,9 @@ input PaymentMethodInput {
 }
 
 input BraintreeInput {
-    payment_method_nonce: String!
-    is_active_payment_token_enabler: Boolean!
-    device_data: String
+    payment_method_nonce: String! @doc(description:"Required input for Braintrees payments")
```

Comment:
Unfortunately, the added description doesn't explain the field purpose.
The `payment_method_nonce` is a one-time payment token generated by Braintree payment gateway based on card details and this token is required to make sale transaction.